Framework/Algorithms/src/MaxEnt.cpp

```diff
@@ -443,7 +456,7 @@ std::vector<double> MaxEnt::move(const QuadraticCoefficients &coeffs,
 
   std::vector<double> delta(dim, 0); // delta at current alpha
 
-  while ((fabs(eps) > chiEps) && (iter < alphaIter)) {
+  while ((fabs(eps / chiTarget) > chiEps) && (iter < alphaIter)) {
```

Comment:
is chiEps a relative value or absolute? You seem to have gone for relative and the old code absolute.

Reply:
It has to be relative because `chiTarget` in this function is a relative quantity in the range 0- 1

Reply:
looking at the code I would have thought it was only alpha that had been constrained

Reply:
Alpha is the Lagrange multiplier for the constraint on the chi squared.